### ISSUE TITLE
Initialize Vue.js environment

### ISSUE THREAD

**Issue by ghilbut** [ghilbut/build-your-first-service.x#1]
- Create Vue.js project
- Apply Vuetify.js with Material Icons
- Apply Login Form

**Comment by ghilbut** [comment]
```bash
$ vue create vue.js
Vue CLI v3.12.1
┌───────────────────────────┐
│  Update available: 4.2.2  │
└───────────────────────────┘
? Please pick a preset: Manually select features
? Check the features needed for your project: Babel, TS, Router, Vuex, Linter, U
nit, E2E
? Use class-style component syntax? Yes
? Use Babel alongside TypeScript (required for modern mode, auto-detected polyfi
lls, transpiling JSX)? Yes
? Use history mode for router? (Requires proper server setup for index fallback 
in production) Yes
? Pick a linter / formatter config: Prettier
? Pick additional lint features: Lint on save, Lint and fix on commit
? Pick a unit testing solution: Jest
? Pick a E2E testing solution: Nightwatch
? Where do you prefer placing config for Babel, PostCSS, ESLint, etc.? In packag
e.json
? Save this as a preset for future projects? No

$ cd vue.js
$ yarn install
$ yarn serve
```

**Comment by ghilbut** [comment]
```
$ vue add vuetify
? Choose a preset: Configure (advanced)
? Use a pre-made template? (will replace App.vue and HelloWorld.vue) Yes
? Use custom theme? No
? Use custom properties (CSS variables)? No
? Select icon font Material Icons
? Use fonts as a dependency (for Electron or offline)? No
? Use a-la-carte components? Yes
? Select locale English

$ yarn install
$ yarn serve
```
